docs/install.md

```diff
@@ -14,15 +14,54 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Install 
+# Installation
 
 There are three ways to use Triton Model Analyzer:
 
-1. Using the `pip`:
+1. Building the Dockerfile:
+
+   The recommended way to use Model Analyzer is with [`docker`](https://docs.docker.com/get-docker/). First, clone the Model Analyzer's git repository,
+   then build the docker image.
+
+   ```
+   $ git clone https://github.com/triton-inference-server/model_analyzer
+   $ docker build --pull -t model-analyzer .
+   ```
+
+   The above command will pull all the containers that model analyzer needs to run. The Model Analyzer's Dockerfile bases the container on the latest `tritonserver` containers from NGC. Now you can run the container with:
+
    ```
-   $ pip3 install git+https://github.com/triton-inference-server/model_analyzer
+   $ docker run -it --privileged --rm --gpus all \
+        -v /var/run/docker.sock:/var/run/docker.sock \
```

Comment:
I think mounting `docker` makes things very complicated (the path issues, etc). If they use Docker they are encouraged to use the `local` or `remote` mode. But if they want to use `docker` mode, they can use `dind`  or mount the `docker.sock`  (as the last option) with the list of issues that they need to be aware of.

Reply:
Does this include if they use the Triton SDK? Is it not the case that the recommended way to use docker in docker is to mount the sock file? We'll be specific about the issues.

Reply:
I think you addressed it correctly in your PR. Thanks!